blueoil/cmd/output_event.py

```diff
@@ -63,32 +74,42 @@ def output(tensorboard_dir, output_dir, metrics_keys, steps, output_file_base="m
 
     columns = [_column_name(event_accumulator, metrics_key)
                for event_accumulator, metrics_key in itertools.product(event_accumulators, metrics_keys)]
-    columns.sort()
-    df = pd.DataFrame([], columns=columns)
+
+    values_step_dict = {}
+    value_matrix = []
+
+    for metrics_key in metrics_keys:
+        if not value_matrix:
+            step_list = sorted(_step_list(event_accumulator, metrics_key), reverse=True)
+            value_matrix.append(step_list)
 
     for event_accumulator in event_accumulators:
         for metrics_key in metrics_keys:
             value_step_list = _value_step_list(event_accumulator, metrics_key)
-            for value, step in value_step_list:
-                column_name = _column_name(event_accumulator, metrics_key)
-                df.loc[step, column_name] = value
+            values_step_dict = dict(value_step_list)
 
-    if steps:
-        df = df[steps, :]
-
-    df = df.sort_index(ascending=False)
-
-    # index to column. and re-order column.
-    df["step"] = df.index
-    df = df[["step"] + columns]
+            for step in step_list:
+                if step not in values_step_dict:
+                    values_step_dict[step] = ''
```

Comment:
Key of `values_step_dict` looks `value`. Not `step`. Correct?
I'm not sure why this line's key is `step`.

Reply:
@iizukak 
I am sorry, it's my mistake, I forget to change line#37 to `(event.value, event.step)`. The sequence is reversed.
Thank you for capturing this significant mistake!